package.json

```diff
@@ -12,7 +12,9 @@
     "prettier": "prettier --write 'src/**/*.js'"
   },
   "dependencies": {
+    "@bumaga/tabs": "^0.2.0",
```

Comment:
It is done now so don't change it, but was there a reason to use such an obscure package? Is it someone you know?

Reply:
Our developer chose @bumaga/tabs because it is a really small and easy to use package to style and use tabs in React.